pynumaflow/function/server.py

```diff
@@ -109,6 +109,7 @@ def __init__(
         self._max_message_size = max_message_size
         self._max_threads = max_threads
         self.cleanup_coroutines = []
+        self.background_tasks = set()
```

Comment:
could you please add a comment around this? please reference the weakref in asyncio